girder_worker/plugins/docker/__init__.py

```diff
@@ -7,7 +11,58 @@ def before_run(e):
         executor.validate_task_outputs(e.info['task_outputs'])
 
 
-def cleanup(e):
+def _read_from_config(key, default):
+    """
+    Helper to read Docker specific config values from the worker config files.
+    """
+    if config.has_option('docker', key):
+        return config.get('docker', key)
+    else:
+        return default
+
+
+def docker_gc(e):
+    """
+    Garbage collect containers that have not been run in the last hour using the
+    https://github.com/spotify/docker-gc project's script, which is copied in
+    the same directory as this file. After that, deletes all images that are
+    no longer used by any containers.
+    """
+    print('Garbage collecting docker containers and images.')
```

Comment:
We should, though that will involve squashing it out across many places. Until now we've just been redirecting stdout to a file, but getting decent logging is important, so we should open an issue for it.